revenue_app/utils.py

```diff
@@ -1,20 +1,40 @@
+import numpy as np
 import pandas as pd
 
 
-# transactions_w_pp = pd.read_excel('datasets/Revenue Queries.xlsx', sheet_name=1, header=1, usecols=range(0,16))
-# transactions_w_email = pd.read_excel('datasets/Revenue Queries.xlsx', sheet_name=2, header=1, usecols=range(0,16))
-
 def get_rollups():
-    return pd.read_excel('datasets/Revenue Queries.xlsx', sheet_name=0, header=1, usecols=range(0,53))
+    return pd.read_excel('datasets/Revenue Queries.xlsx', sheet_name=0, header=1, usecols=range(0, 53))
 
 
 def get_transactions():
-    return pd.read_csv('datasets/transactions.csv')
+    transactions = pd.read_csv('datasets/transactions.csv').replace(np.nan, '', regex=True)
+    if 'sale__eb_tax__epp__1' in transactions.columns:
+        transactions.drop('sale__eb_tax__epp__1', axis=1, inplace=True)
+    transactions['transaction_created_date'] = pd.to_datetime(
+        transactions['transaction_created_date'],
+        format="%m/%d/%Y"
+    )
+    return transactions
 
 
 def get_organizer_sales():
-    return pd.read_csv('datasets/organizer_sales.csv')
+    organizer_sales = pd.read_csv('datasets/organizer_sales.csv').replace(np.nan, '', regex=True)
+    if 'organizer_email' in organizer_sales.columns:
+        organizer_sales.rename(columns={'organizer_email': 'email'})
+    return organizer_sales
 
 
 def get_organizer_list():
     return []
+
+
+def filter_transactions_by_date(start_date, end_date=None, df=None):
+    '''dates YYYY-MM-DD'''
```

Comment:
This docstring doesn't seem all that useful. Can we improve it? Also, PEP8 suggests using triple-double quotes in here rather than single quotes. See https://www.python.org/dev/peps/pep-0257/#id15

Reply:
Oh! I thought I removed it. That should go